mlops/bicep/vnet_publicip_sandbox_setup.bicep

```diff
@@ -161,12 +170,15 @@ module silos './modules/fl_pairs/vnet_compute_storage_pair.bicep' = [for i in ra
     pairRegion: siloRegions[i]
     tags: tags
 
-    pairBaseName: '${demoBaseName}-silo${i}-${siloRegions[i]}'
+    pairBaseName: '${demoBaseName}-silo${i}'
 
-    computeName: 'cpu-silo${i}-${siloRegions[i]}' // let's not use demo base name
-    computeSKU: computeSKU
+    compute1Name: 'silo${i}-01' // let's not use demo base name in cluster name
```

Comment:
could we keep the region here? it looks much nicer

proposal: `silo${i}-${siloRegions[i]}-1`

Reply:
I removed the region name to avoid one additional step. Users can have different region-quota limits. If a user changes regions while provisioning resources, the respective names need to be replaced(YAML files) if the region is a part of the compute/datastore name. Please let me know if you feel otherwise.

Reply:
Yes you are right if someone needs to adapt the provisioning, they might have to tweak their config files. But what I loved about the previous names is that out-of-the-box effect of our demo, showing regional names in computes, and in the metrics. We could figure out a way to address that in another setting somewhere else though.

Reply:
As discussed, I'll create another PR to accommodate these changes.